lib/internal/errors.js

```diff
@@ -85,4 +85,33 @@ module.exports = exports = {
 //
 // Note: Please try to keep these in alphabetical order
 E('ERR_ASSERTION', (msg) => msg);
+E('ERR_INVALID_ARG_TYPE', invalidArgType);
+E('ERR_INVALID_CALLBACK', 'callback must be a function');
+E('ERR_STDERR_CLOSE', 'process.stderr cannot be closed');
+E('ERR_STDOUT_CLOSE', 'process.stdout cannot be closed');
+E('ERR_UNK_STDIN_TYPE', 'Unknown stdin file type');
```

Comment:
https://github.com/nodejs/node/pull/11298 spells the full `UNKNOWN`, this should do the same (TBH I can not understand what `UNK` stands for at first glance...).